Sources/HaCWebsiteLib/ViewModels/ImageHero.swift

```diff
@@ -18,12 +38,36 @@ struct ImageHero {
     case color(String)
   }
 
+  enum Color {
```

Comment:
I feel like the name of this is a bit misleading. I'm not entirely sure what would be better though, but currently, this deals with tones not colours if I've understood it right.

Reply:
Agreed this is a little unclear`BackgroundColourType` might be more meaningful